lib/ultimate_turbo_modal/base.rb

```diff
@@ -21,8 +21,11 @@ def initialize(
     footer_divider: UltimateTurboModal.configuration.footer_divider,
     header: UltimateTurboModal.configuration.header,
     header_divider: UltimateTurboModal.configuration.header_divider,
+    title_tag: UltimateTurboModal.configuration.title_tag,
     padding: UltimateTurboModal.configuration.padding,
-    request: nil, title: nil
+    request: nil,
+    title: nil,
+    bootstrap_modal_size: UltimateTurboModal.configuration.bootstrap_modal_size
```

Comment:
This should not be on the base class if it's meant for the Bootstrap variant only. Should be in the Bootstrap subclass.